lib/PersistedPaneset/PersistedPaneset.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useLocalStorage, writeStorage } from '@rehooks/local-storage';
+
+import { Paneset } from '@folio/stripes-components';
+
+const propTypes = {
+  appId: PropTypes.string.isRequired,
+  children: PropTypes.node,
+  id: PropTypes.string.isRequired,
+};
+
+const PersistedPaneset = ({
+  appId,
+  children,
+  id: panesetId,
+}) => {
+  const persistKey = `${appId}/${panesetId}/layout`;
+  const [storedPanesetLayout] = useLocalStorage(persistKey);
+
+  return (
+    <Paneset
```

Comment:
I that original `Paneset` can accept more props than specified here like `isRoot`, `nested` and whatever props in the future. For the sake of completeness, what do you think about adding the possibility to pass other props?